src/operator/mod.rs

```diff
@@ -7,9 +7,11 @@ pub mod physical_scan;
 pub mod physical_topn;
 
 use crate::metadata::MdAccessor;
+use crate::property::PhysicalProperties;
 use crate::statistics::Statistics;
 use std::any::Any;
 use std::rc::Rc;
+use std::todo;
```

Comment:
can be deleted, and compilations can also pass

Reply:
done